test/test.go

```diff
@@ -0,0 +1,95 @@
+package test
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/NethermindEth/1click/internal/pkg/commands"
+)
+
+func CreateFakeDep(t *testing.T, dependency string) (depPath string) {
+	depPath = t.TempDir()
+	file, err := os.Create(filepath.Join(depPath, dependency))
+	if err != nil {
+		t.Fatalf("Can't fake dependency %s", dependency)
+	}
+	file.Close()
+
+	PATH := os.Getenv("PATH")
+	err = os.Setenv("PATH", fmt.Sprintf("%s:%s", PATH, depPath))
+	if err != nil {
+		t.Fatalf("Can't fake dependency %s", dependency)
+	}
+	return
+}
+
+func DeleteFakeDep(depPath string) {
+	PATH := os.Getenv("PATH")
+	PATH = strings.ReplaceAll(PATH, ":"+depPath, "")
+	os.Setenv("PATH", PATH)
+}
+
+type SimpleCMDRunner struct {
```

Comment:
Add struct's description or summary. The name itself does not say that this is a stub, or a stub or mock of what struct. So the comments should say so